code/modules/clothing/head/misc.dm

```diff
@@ -21,6 +21,7 @@
 	icon_state = "tophat"
 	item_state = "that"
 	dog_fashion = /datum/dog_fashion/head
+	throwforce = 1

```

Comment:
make the user take double electricity damage if they touch it while on a powered wire.